setup.py

```diff
@@ -1,14 +1,13 @@
 from setuptools import setup, find_packages
 
 setup(
-      name='bullet',
-      version='2.0.0',
-      description='Beautiful Python prompts made simple.',
-      long_description="Extensive support for Python list prompts \
-            formatting and colors",

```

Comment:
I actually removed the `\` and made this a single line string manually. This was the only change that I made manually.